target/portfolio-1/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
```

Comment:
Have you run css/html/js format check to clean the format?
Try to add an automatic  format check in your github repo to make sure every pr is clean.

Reply:
I'm looking on the action tab to see how to add the format checks and the automatic deploy whenever a branch is merged with master. If I'm still having trouble finding it, I will definitely reach out to the chat!